catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/view/NodeModal.view.js

```diff
@@ -70,7 +70,11 @@ define([
                 this.listenTo(wreqr.vent, 'fieldErrorChange:' + this.model.contentInfo.get('segmentId'), this.updateContentTabError);
 
                 this.model.generalInfo.get('segments').forEach(function (segment){
-                    segment.set('identityNode', this.model.get('identityNode'));
+                    if(this.model.get('summary').constructor === Object){
```

Comment:
Fixes an issue where additional local nodes can't be added because they aren't backbone models.